fcrepo-export/src/main/java/org/fcrepo/exporter/Config.java

```diff
@@ -108,14 +108,23 @@ public File getDescriptionDirectory() {
      * Sets the URI of the resource to import/export
      */
     public void setResource(final String resource) throws URISyntaxException {
-        this.resource = new URI(resource);
+        if (resource != null && (resource.lastIndexOf('/') == resource.length() - 1)) {
```

Comment:
And maybe have this first `setResource` call the second `setResource(URI)`, and consolidate this substring logic in just one method.